lib/router.js

```diff
@@ -45,18 +46,26 @@ router.get('/slashCommands/ci', (req, res) => {
 router.get('/slashCommands/ext', (req, res) => {
     const command = req.query && req.query.command;
     const text = req.query.text;
+    const update = req.query.update;
```

Comment:
That's actually the basic Slack API documentation: https://api.slack.com/slash-commands
All custom slash commands send only two parameters: the typed command itself and the rest of the text after it, so `/ext arg1 arg2 arg3 arg4`, `req.query.text` will be equals to `arg1 arg2 arg3 arg4`, just a plain string. There's no way to customize how Slack send us that data, so we have to parse it all from `req.query.text`. This has to be like that regardless of what we decide about "update + later searching" or "update + search at the same time". On that matter, I'll post a separate comment in the PR's main page, ok?